src/tslint.ts

```diff
@@ -84,7 +85,8 @@ module Lint {
             return {
                 failureCount: failures.length,
                 format: this.options.formatter,
-                output: output
+                output: output,
+                failures: failures
```

Comment:
we now alphabetize keys in object literals, and this is enforced by tslint, which is why your build is failing